src/contracts/types/UserCallTypes.sol

```diff
@@ -15,6 +15,7 @@ struct UserOperation {
     uint256 deadline;
     address dapp; // nested "to" for user's call
     address control; // address for preOps / validation funcs
+    uint32 callConfig;
     address sessionKey;
     bytes data;
     bytes signature;
```

Comment:
We need to add `callConfig` to:
- `USER_TYPEHASH` constant, in this same file
- `_getUserOpHash` function, in `AtlasVerification.sol`

Reply:
Good catch! Fixed in next commit